docs/en/administrator-guide/config/fe_config.md

```diff
@@ -261,6 +267,12 @@ This configuration can play a role in certain scenarios. Assume that the initial
 
 ### `enable_auth_check`
 
+### `enable_check_data_quota_on_load`
```

Comment:
1. I think check database quota can be checked upon every load. So we can check every time without this config.
2. How about data quota update frequency previously?  It's used by table report, It seems that we can continue to use it or not ?

Reply:
@chaoyli 
1. The main consideration is to make the user's load task not fail if they don't want use this load quota limit, but now it seems a bug?
2. TabletStatMgr will update replica rowcount and datasize every 5min now, but not compute data quota, data quota will be computed only when show data operation is executed or create table or alter table schema.